app/Http/Transformers/PhotoTransformer.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use Rogue\Models\Photo;
+use League\Fractal\TransformerAbstract;
+
+class PhotoTransformer extends TransformerAbstract
+{
+    /**
+     * List of resources to automatically include
+     *
+     * @var array
+     */
+    protected $defaultIncludes = [
+        'event',
+    ];
+
+    /**
+     * Transform resource data.
+     *
+     * @param \Rogue\Models\Photo $photo
+     * @return array
+     */
+    public function transform(Photo $photo)
+    {
+        return [
+            'event_id' => $photo->event_id,
+            'signup_id' => $photo->signup_id,
+            'northstar_id' => $photo->northstar_id,
+            'campaign_id' => $photo->signup->campaign_id,
+            'campaign_run_id' => $photo->signup->campaign_run_id,
+            'post' => [
```

Comment:
@DFurnes I definitely like the idea of having a standard `PostTransformer` . I am going back and forth on each post type having it's own endpoint. It would certainly make things a little more straightforward because we would never have to do the work of figuring out what we trying to store/grab/return. But then at the same time I feel like all of these things are really the same, it's just they store different stuff, so I like just having one endpoint to deal with them. 

It's also weird because right now we only have photos so I might have shot myself in the foot with too much future thinking.